go.mod

```diff
@@ -2,17 +2,17 @@ module github.com/ipfs/go-mfs
 
 require (
 	github.com/ipfs/go-blockservice v0.1.1
-	github.com/ipfs/go-cid v0.0.2
+	github.com/ipfs/go-cid v0.0.7
 	github.com/ipfs/go-datastore v0.0.5
 	github.com/ipfs/go-ipfs-blockstore v0.0.1
 	github.com/ipfs/go-ipfs-chunker v0.0.1
 	github.com/ipfs/go-ipfs-exchange-offline v0.0.1
 	github.com/ipfs/go-ipfs-util v0.0.1
 	github.com/ipfs/go-ipld-format v0.0.2
-	github.com/ipfs/go-log v0.0.1
-	github.com/ipfs/go-merkledag v0.1.0
+	github.com/ipfs/go-log v1.0.5
+	github.com/ipfs/go-merkledag v0.2.3
 	github.com/ipfs/go-path v0.0.7
-	github.com/ipfs/go-unixfs v0.1.0
+	github.com/ipfs/go-unixfs v0.2.7-0.20211112011223-bd53b6a811b1
```

Comment:
TODO: Update to released go-unixfs before merging. cc @guseggert once we do this we're ready to merge here.